ui/imports/shared/stores/TransactionStore.qml

```diff
@@ -14,10 +14,11 @@ QtObject {
 
     property var mainModuleInst: mainModule
     property var walletSectionTransactionsInst: walletSectionTransactions
+    property var walletSectionSendInst: walletSectionSend
 
     property string currentCurrency: walletSection.currentCurrency
     property var allNetworks: networksModule.all
-    property var accounts: walletSectionAccounts.model
+    property var accounts: walletSectionSend.accounts
```

Comment:
`walletSectionSendInst`

Reply:
And all the changed occurrences below, otherwise `walletSectionSend` can be `nullish` on startup and will generate lots of warnings